frontend/integration-tests/tests/olm/global-installmode.scenario.ts

```diff
@@ -170,15 +169,6 @@ describe('Interacting with an `AllNamespaces` install mode Operator (Redis)', ()
     expect(crudView.successMessage.getText()).toContain(`${redisEnterpriseCluster} has been updated to version`);
   });
 
-  it('displays Kubernetes objects associated with the `RedisEnterpriseCluster` in its "Resources" section', async() => {
```

Comment:
This test is the one that needs to be readded once the RedisCluster is adjusted - currently there are no kubernetes objects being created to display in the resources section so this test was not failing because of my change - my change actually highlighted the issue. It had not been effectively testing this scenario before hand.